diff --git a/node_modules/gatsby-plugin-algolia-docsearch/gatsby-ssr.js b/node_modules/gatsby-plugin-algolia-docsearch/gatsby-ssr.js
index aa06cf2..9006c27 100644
--- a/node_modules/gatsby-plugin-algolia-docsearch/gatsby-ssr.js
+++ b/node_modules/gatsby-plugin-algolia-docsearch/gatsby-ssr.js
@@ -9,7 +9,7 @@ var _react = _interopRequireDefault(require("react"));
 var _commonTags = require("common-tags");
 
 function _templateObject() {
-  var data = (0, _taggedTemplateLiteralLoose2["default"])(["\n        var observer = new MutationObserver(function (mutations, instance) {\n          var docuSearchElem = document.querySelector('", "');\n          if (docuSearchElem) {\n            docsearch({\n              apiKey: \"", "\",\n              indexName: \"", "\",\n              inputSelector: \"", "\",\n              debug: ", "\n            });\n            instance.disconnect(); // stop observing\n            return;\n          }\n        });\n\n        // start observing\n        document.addEventListener(\"DOMContentLoaded\", function() {\n          observer.observe(document, {\n            childList: true,\n            subtree: true\n          });\n        });\n        "]);
+  var data = (0, _taggedTemplateLiteralLoose2["default"])(["\n        var observer = new MutationObserver(function (mutations, instance) {\n          var docuSearchElem = document.querySelector('", "');\n          if (docuSearchElem) {\n            docsearch({\n              apiKey: \"", "\",\n              appId: \"",  "\",\n              indexName: \"", "\",\n              inputSelector: \"", "\",\n              debug: ", "\n            });\n            instance.disconnect(); // stop observing\n            return;\n          }\n        });\n\n        // start observing\n        document.addEventListener(\"DOMContentLoaded\", function() {\n          observer.observe(document, {\n            childList: true,\n            subtree: true\n          });\n        });\n        "]);
 
   _templateObject = function _templateObject() {
     return data;
@@ -22,6 +22,7 @@ exports.onRenderBody = function (_ref, _ref2) {
   var setHeadComponents = _ref.setHeadComponents,
       setPostBodyComponents = _ref.setPostBodyComponents;
   var apiKey = _ref2.apiKey,
+      appId = _ref2.appId,
       indexName = _ref2.indexName,
       inputSelector = _ref2.inputSelector,
       _ref2$debug = _ref2.debug,
@@ -44,7 +45,7 @@ exports.onRenderBody = function (_ref, _ref2) {
     key: "plugin-docsearch-initiate",
     type: "text/javascript",
     dangerouslySetInnerHTML: {
-      __html: (0, _commonTags.stripIndent)(_templateObject(), inputSelector, apiKey, indexName, inputSelector, debug === true ? "true" : "false")
+      __html: (0, _commonTags.stripIndent)(_templateObject(), inputSelector, apiKey, appId, indexName, inputSelector, debug === true ? "true" : "false")
     }
   })]);
 };
\ No newline at end of file
